src/web-server/html-formatter.js

```diff
@@ -57,72 +57,89 @@ const formatRow = (
 };
 
 const footerClose = '<address>nohost/0.0.2 (Web)</address></body></html>';
+export default class {
```

Comment:
Don't bother with a `class` if not holding state.  Can be an object or just expose the functions.

Reply:
It still seems to me like they are supposed to be grouped somehow, since it will be easier to swap formatter objects instead of individual functions.